demo/query_known/query_known.md

```diff
@@ -158,6 +158,154 @@ with GUAC to store SBOM and SLSA attestations for quick access. For now, we can
 quickly locate the SBOM but we also learn that we are missing `hasSLSA`
 attestations for this package.
 
+Along the same lines, we have the hash of an executable binary that was
```

Comment:
Does this also have to be updated on the documentation repo? If so, should we start removing this from here?

Reply:
yea - id say we can update everything but this file in the PR and then apply the diff on the reviewed one later.

Reply:
the rest seem to be formatting, so let's just keep this issue open for now